stanford/playbooks/sandbox.yml

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_ASSIGN_PUBLIC_IP: yes
+    CLUSTER_INSTANCE_TAG_NAME: "{{ COMMON_DEPLOYMENT }}-box-{{ CLUSTER_NAME }}-{{ CLUSTER_SANDBOX_NAME }}"
+    CLUSTER_INSTANCE_TYPE: t2.medium
+    CLUSTER_NAME: sandbox
+    CLUSTER_NETWORK: public
+    CLUSTER_SANDBOX_NAME: master
+  when: redeploy | default(False) | bool
+# This is largely cribbed from `playbooks/roles/edx_sandbox.yml`
+- name: Configure instance(s)
+  hosts:
+    - 'cluster_group_launched'
+    - "tag_Name_{{ COMMON_DEPLOYMENT }}-box-sandbox-{{ CLUSTER_SANDBOX_NAME }}"
+  become: True
+  gather_facts: True
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+  vars:
```

Comment:
Everything from here down to the end of the file is identical to the copy in `playbooks/roles/edx_sandbox.yml`